docs/reference/tab-widgets/data-tiers.asciidoc

```diff
@@ -0,0 +1,90 @@
+// tag::cloud[]
+. Log in to the {ess-trial}[{ess} Console].
+
+. Add or select your deployment from the {ess} home page or the deployments
+page.
+
+. From your deployment menu, select **Edit deployment**.
+
+. To enable a data tier, click **Add capacity**.
+
+experimental:[] **Frozen tier** 
+
+The frozen tier is not yet available on {ess}. However, you can follow these
+steps to effectively recreate a frozen tier in your deployment:
+
+. Choose an existing tier to use. You'll typically use the cold tier, but the
+hot and warm tiers are also supported. You can use this tier as a shared tier,
+or use it exclusively as a frozen tier.
+
+. On the **Edit deployment** page, click **Edit elasticsearch.yml** for your
+chosen tier.
+
+. In the `elasticsearch.yml` configuration, set
+<<searchable-snapshots-shared-cache,`xpack.searchable.snapshot.shared_cache.size`>>
+to up to 90% of available disk space. The tier uses this space to
+create a <<shared-cache,shared, fixed-size cache>> for
+<<searchable-snapshots,searchable snapshots>>.
++
+[source,yaml]
+----
+xpack.searchable.snapshot.shared_cache.size: 50GB
+----
+
+. Click **Save** and **Confirm** to apply your changes.
+
+**Enable autoscaling**
+
+{cloud}/ec-autoscaling.html[Autoscaling] automatically adjusts your deployment's
+capacity to meet your storage needs. To enable autoscaling, select **Autoscale
+this deployment** on the **Edit deployment** page. Autoscaling is only available
+for {ess}.
+// end::cloud[]
+
+// tag::self-managed[]
+To assign a node to a data tier, add the respective <<node-roles,node role>> to
+the node's `elasticsearch.yml` file. Changing an existing node's roles requires
```

Comment:
This feels weird right after the Cloud-specific stuff, almost like this is something a user will need to set in their cloud deployment also (maybe it just looks like this in the raw text rather than the rendered version).

Reply:
Yeah. It's hard to tell from the source, but this content actually displays in a completely separate tab. It should be clear that it doesn't apply to ESS at all.

Sorry that the relationship isn't a bit more clear in the source. We had to jump through a few hoops to get the tabbed display. It may be clearer in the preview: https://elasticsearch_71195.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/use-elasticsearch-for-time-series-data.html

<img width="757" alt="Screen Shot 2021-04-01 at 6 31 52 PM" src="https://user-images.githubusercontent.com/40268737/113360877-a748db00-9318-11eb-872d-3f971b61380c.PNG">